interactive_engine/executor/ir/core/src/glogue/pattern.rs

```diff
@@ -820,6 +872,77 @@ fn has_expr_eq(expr: &common_pb::Expression) -> bool {
     false
 }
 
+/// Pick up EdgeExpand Operator from a binder in pb::Pattern's sentences
+fn get_edge_expand_from_binder<'a, 'b>(
+    binder: &'a pb::pattern::Binder, edge_id: PatternId,
+    edge_data_map: &'b mut BTreeMap<PatternId, PbEdgeOrPath>,
+) -> IrPatternResult<&'a pb::EdgeExpand> {
+    use pb::pattern::binder::Item as BinderItem;
+    if let Some(BinderItem::Path(path_expand)) = binder.item.as_ref() {
+        let hop_range = path_expand
+            .hop_range
+            .as_ref()
+            .ok_or(ParsePbError::EmptyFieldError("pb::PathExpand::hop_range".to_string()))?;
+        if hop_range.lower < 1 {
+            // The path with range from 0 cannot be translated to oprs that can be intersected.
+            return Err(IrPatternError::Unsupported(format!(
+                "PathExpand in Pattern with lower range of {:?}",
+                hop_range.lower
+            )))?;
+        }
+        let expand_base = path_expand
+            .base
+            .as_ref()
+            .ok_or(ParsePbError::EmptyFieldError("PathExpand::base in Pattern".to_string()))?;
+        expand_base
+            .get_v
+            .as_ref()
+            .ok_or(IrPatternError::Unsupported(
+                "PathExpand::base in Pattern doesn't have GetV".to_string(),
+            ))?;
```

Comment:
Do not treat it as an error. Currently, in Cypher, it will be  ExpandE+GetV in PathExpandBase; while in Gremlin, it will be ExpandV in PathExpandBase.

Reply:
Fixed